metricbeat/mb/registry.go

```diff
@@ -121,6 +123,7 @@ type ModulesSource interface {
 	HasMetricSet(module, name string) bool
 	MetricSetRegistration(r *Register, module, name string) (MetricSetRegistration, error)
 	ModulesInfo(r *Register) string
+	ProcessorsForMetricSet(r *Register, moduleName string, metricSetName string) (*processors.Processors, error)
```

Comment:
Nit. Other methods in this interface use `module` for the module name, and `name` for the metricset name.
```suggestion
	ProcessorsForMetricSet(r *Register, module string, name string) (*processors.Processors, error)
```

Not sure what option is better though... but it should be consistent.

Reply:
Fixed, changed to a form consistent with other methods.